pkg/dotnet/core_deps/parse_test.go

```diff
@@ -15,19 +15,251 @@ import (
 
 func TestParse(t *testing.T) {
 	vectors := []struct {
-		file    string // Test input file
-		want    []types.Library
-		wantErr string
+		file     string // Test input file
+		wantLibs []types.Library
+		wantDeps []types.Dependency
+		wantErr  string
 	}{
+		{
+			file: "testdata/MyExample.deps.json",
+			wantLibs: []types.Library{
```

Comment:
We try to make test cases smaller for better support.
Can you use a library with fewer dependencies and less depth?